src/Pages/new/GetVerified.tsx

```diff
@@ -0,0 +1,73 @@
+import { FormattedMessage } from "react-intl";
+import { useNavigate } from "react-router-dom";
+
+import { services } from "Pages/Verification";
+import Nip5Service from "Element/Nip5Service";
+import ProfileImage from "Element/ProfileImage";
+
+import messages from "./messages";
+
+export default function GetVerified() {
+  const navigate = useNavigate();
+
+  const onNext = async () => {
+    navigate("/new/import");
+  };
+
+  return (
+    <div className="main-content new-user">
+      <div className="progress-bar">
+        <div className="progress progress-third"></div>
+      </div>
+      <h1>
+        <FormattedMessage {...messages.Identifier} />
+      </h1>
+      <h4>
+        <FormattedMessage {...messages.PreviewOnSnort} />
+      </h4>
+      <div className="profile-preview-nip">
+        <ProfileImage pubkey="63fe6318dc58583cfe16810f86dd09e18bfd76aabc24a0081ce2856f330504ed" />
```

Comment:
Its possible to pass the current user pubkey here instead along with an nip5 override? 

We also should force validation so it always shows as valid

User can then enter their handle and preview the profile with the nip5